### PR TITLE
Move makemkv progress into own logs

### PR DESCRIPTION
**Pull request by 1337-server**
# Description
Moves the makeMKV progress logs into its own log as with UHD and large bluray logs become large and annoying to read

Fixes # (issue title here)

## Type of change
Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.
Please also list any relevant details for your test configuration

- [x] Docker
- [x] Other (Please state here)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have tested that my fix is effective or that my feature works

# Changelog:

Include the details of changes made here
- Change 1
- Change 2

# Logs
Attach logs from successful test runs here

### CODE REVIEW

**Comment by shitwolfymakes** [comment]
Rather than redirecting these to a new log file, can we create a MakeMKV/Hanbrake "level" of logging, below even debug?

If that's not possible, we could always make including the logs in the regular logfile a setting in arm.yaml

**Comment by 1337-server** [comment]
> Rather than redirecting these to a new log file, can we create a MakeMKV/Hanbrake "level" of logging, below even debug?
> 
> If that's not possible, we could always make including the logs in the regular logfile a setting in arm.yaml

The problem with this is that these progress logs are needed for keeping track of progress/progress bar on the homepage. Without them it just shows the current status and not the ETA or job progress.

**Comment by shitwolfymakes** [comment]
> The problem with this is that these progress logs are needed for keeping track of progress/progress bar on the homepage. Without them it just shows the current status and not the ETA or job progress.

Ah, shit. Is that how you got that working? I would image doing it that way was your last resort?

**Comment by 1337-server** [comment]
It's the only way I can think of doing it. I'm not sure if we have any other option, as there is no update output other than the progress logs.
The alternative at the time was to tail the current log and wait for updates. But I hated that and wanted updates on the homepage.

**Comment by 1337-server** [comment]
This is ready for review.

File: arm/ripper/makemkv.py [hunk 1]
```diff
@@ -104,7 +108,9 @@ def process_single_tracks(job, logfile, rawpath):
             filepathname = os.path.join(rawpath, track.filename)
             logging.info(f"Ripping title {track.track_number} to {shlex.quote(filepathname)}")
 
-            cmd = f'makemkvcon mkv {job.config.MKV_ARGS} -r --progress=-stdout --messages=-stdout ' \
+            cmd = f'makemkvcon mkv {job.config.MKV_ARGS} -r ' \
```
Comment (shitwolfymakes) [inline arm/ripper/makemkv.py:111]: Please add a comment explaining what this command does

File: arm/ui/json_api.py [hunk 2]
```diff
@@ -91,12 +91,13 @@ def percentage(part, whole):
     return percent
 
 
-def process_makemkv_logfile(logfile, job, job_results):
+def process_makemkv_logfile(job, job_results):
     """
     Process the logfile and find current status\n
```
Comment (shitwolfymakes) [inline arm/ui/json_api.py:96]: Update docstring, and clarify how this function works

File: arm/ripper/logger.py [hunk 3]
```diff
@@ -78,11 +78,14 @@ def clean_up_logs(logpath, loglife):
     now = time.time()
     logging.info(f"Looking for log files older than {loglife} days old.")
 
-    for filename in os.listdir(logpath):
-        fullname = os.path.join(logpath, filename)
-        if fullname.endswith(".log") and os.stat(fullname).st_mtime < now - loglife * 86400:
-            logging.info(f"Deleting log file: {filename}")
-            os.remove(fullname)
+    logs_folders = [logpath, os.path.join(logpath, 'progress')]
+    for log_dir in logs_folders:
```
Comment (shitwolfymakes) [inline arm/ripper/logger.py:82]: Explain what this loop does

**Review by shitwolfymakes — CHANGES_REQUESTED** [review]

File: arm/ripper/makemkv.py [hunk 4]
```diff
@@ -55,7 +55,9 @@ def makemkv(logfile, job):
     # Rip bluray
     if (job.config.RIPMETHOD == "backup" or job.config.RIPMETHOD == "backup_dvd") and job.disctype == "bluray":
         # backup method
-        cmd = f'makemkvcon backup --minlength={job.config.MINLENGTH} --decrypt {job.config.MKV_ARGS} ' \
+        cmd = f'makemkvcon backup --decrypt {job.config.MKV_ARGS} --minlength={job.config.MINLENGTH}' \
```
[the remark this answers is not in the document]
Comment (1337-server) [reply arm/ripper/makemkv.py:58]: The comment is right above it ?

File: arm/ripper/makemkv.py [hunk 5]
```diff
@@ -65,7 +67,9 @@ def makemkv(logfile, job):
 
         # if no maximum length, process the whole disc in one command
         if int(job.config.MAXLENGTH) > 99998:
-            cmd = f'makemkvcon mkv {job.config.MKV_ARGS} -r --progress=-stdout --messages=-stdout ' \
+            cmd = f'makemkvcon mkv {job.config.MKV_ARGS} -r ' \
```
[the remark this answers is not in the document]
Comment (1337-server) [reply arm/ripper/makemkv.py:70]: Line 68

**Comment by 1337-server** [comment]
Ready for re-review.

**Pull request merged**